unicoremc/models.py

```diff
@@ -396,6 +399,20 @@ def create_webhook(self, access_token):
             raise exceptions.AccessTokenRequiredException(
                 'access_token is required')
 
+    def create_unicore_distribute_repo(self):
+        post_data = {
+            "repo_url": self.repo_git_url
```

Comment:
@Rizziepit this is the full git repo url e.g `https://github.com/universalcore/unicore-cms-content-gem-tz-qa.git` the `repo_name` is just the last part `unicore-cms-content-gem-tz-qa`